### PR TITLE
Much FindBugs fixes!

### PR DESCRIPTION
**Pull request by mathieupost**

### CODE REVIEW

**Comment by Delfil** [comment]
All ok

File: src/main/java/tudelft/ti2806/pl3/data/graph/GraphDataRepository.java [hunk 1]
```diff
@@ -155,9 +156,8 @@ public void parseGraph(File nodesFile, File edgesFile, GeneData geneData) throws
 	 */
 	public Map<Integer, DataNode> parseNodes(File nodesFile, Map<String, Genome> genomeMap,
 			GeneData geneData) throws FileNotFoundException {
-		BufferedReader br = new BufferedReader(new InputStreamReader(
-				new BufferedInputStream(new FileInputStream(nodesFile))));
-		Map<Integer, DataNode> nodes = new HashMap<Integer, DataNode>();
+		BufferedReader br = new BufferedReader(new InputStreamReader(new FileInputStream(nodesFile), StandardCharsets.UTF_8));
```
Comment (borismattijssen) [inline src/main/java/tudelft/ti2806/pl3/data/graph/GraphDataRepository.java]: line longer then 120 chars?

File: src/main/java/tudelft/ti2806/pl3/data/wrapper/util/SpaceWrapUtil.java [hunk 2]
```diff
@@ -169,12 +170,17 @@ public static WrappedGraphData collapseGraph(WrappedGraphData original) {
...
-			List<Wrapper> nodeList = new ArrayList<Wrapper>(
+			List<Wrapper> nodeList = new ArrayList<>(
 					bucket.getSecond());
 			for (int i = bucket.getSecond().size() - 1; i > 0; i--) {
 				nodeList.remove(nodeList.size() - 1);
-				candidateList.add(newCandidatePair(bucket.getSecond().get(i),
-						nodeList.get(nodeList.size() - 1)));
+				try {
+					Pair<Integer, Pair<Wrapper, Wrapper>> candidatePair =
+							newCandidatePair(bucket.getSecond().get(i), nodeList.get(nodeList.size() - 1));
```
Comment (borismattijssen) [inline src/main/java/tudelft/ti2806/pl3/data/wrapper/util/SpaceWrapUtil.java]: *Auto-generated comment*

    Line is longer than 120 characters (found 135).

Comment (mathieupost) [inline src/main/java/tudelft/ti2806/pl3/data/wrapper/util/SpaceWrapUtil.java]: Niet waar! Zijn er 108.

**Comment by mathieupost** [comment]
There are still some findbugs results, but most of them are fixed in other branches.

**Comment by kwendel** [comment]
many fixed much good!

**Comment by borismattijssen** [comment]
All guci

**Pull request merged**